.github/workflows/test.yml

```diff
@@ -28,32 +26,40 @@ jobs:
       - name: Set up Go
         uses: actions/setup-go@v4
         with:
-          go-version-file: '${{ github.workspace }}/go.mod'
+          go-version-file: '${{ github.workspace }}/x/go.mod'
```

Comment:
Are we going to use `/x/go.mod` instead of `/go.mod`? What if they are different?

Reply:
I picked x/ because it depends on /, so its version will have to be the most recent one.

Right now, Psiphon forces x/ to be 1.21, but / doesn't need to be